cmd/edv-rest/startcmd/start.go

```diff
@@ -874,14 +874,14 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if r.RequestURI == createVaultPath || r.RequestURI == healthCheckPath {
+	s := strings.SplitAfter(r.RequestURI, "/")
+
+	if len(s) < 3 || r.RequestURI == createVaultPath || r.RequestURI == healthCheckPath {
```

Comment:
Do you need to check for all three conditions, or would `len(s) < 3` suffice?

Reply:
correct. I change it